docs/graphql/manual/auth/authentication/unauthenticated-access.rst

```diff
@@ -31,21 +31,29 @@ A guide on setting up unauthenticated user permissions can be found :ref:`here <
 Configuring unauthenticated / public access
 -------------------------------------------
 
-Depending on your auth setup an unauthenticated role can be configured as follows:
+An unauthenticated role can be configured for the following scenarios: webhook auth setup, JWT auth setup or no auth setup.
 
 Webhooks
 ^^^^^^^^
 
 For :ref:`webhook authentication <auth_webhooks>`, an unauthenticated request is any request for which the webhook returns a ``401 Unauthorized`` response.
 
-For unauthenticated access, you can return a ``200`` status response with your defined unauthenticated role, e.g: ``{ "x-hasura-role": "<anonymous-role>" }``.
+To allow unauthenticated access, the auth webhook should return a ``200`` status response with your unauthenticated role in the headers. For details on the webhook response, refer to :ref:`this section <webhook_response>`.
 
 JWT
 ^^^
 
 For :ref:`JWT authentication <auth_jwt>`, an unauthenticated request is any request which does not contain a JWT token.
 
-You can use the env variable ``HASURA_GRAPHQL_UNAUTHORIZED_ROLE`` or ``--unauthorized-role`` flag to set a role
+You can use the env variable ``HASURA_GRAPHQL_UNAUTHORIZED_ROLE`` or the ``--unauthorized-role`` flag to set a role
 for unauthenticated (non-logged in) users. See :ref:`server_flag_reference` for more details
 on setting this flag/env var.
 
+No auth setup
+^^^^^^^^^^^^^
+
+With no auth setup, an unauthenticated request is one that doesn't set an explicit role defined in permissions.
```

Comment:
@sezgi @rikinsk I'm not quite sure if this is the best definition for an unauthenticated request with no auth setup but couldn't come up with anything else. You have any thoughts on this?

Reply:
I'm not sure how this works. If no role is defined in permissions, what would the env variable be matching? I'll defer to @rikinsk and @tirumaraiselvan on this one.

Reply:
With no auth setup, every request without the admin-secret header is considered as an unauthenticated request. 

Nitpick, maybe moving "no auth setup" as the first section (befoore Webhooks) would make for a more logical reading order